### PR TITLE
feat(Discohook): add presence

### PR DESCRIPTION
**Pull request by NinjaLabs-Dev**
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Picture of presence working

</details>

### CODE REVIEW

File: websites/D/Discohook/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.7",
+  "service": "Discohook",
+  "author": {
+    "id": "150987570715688960",
+    "name": "Ninja#4321"
```
Comment (theusaf) [inline websites/D/Discohook/metadata.json:6]: The discriminator is a mistake in the `pmd` command.
```suggestion
    "name": "Ninja"
```

**Comment by NinjaLabs-Dev** [comment]
Created from request - [Issue](https://github.com/PreMiD/Presences/issues/6117)

File: websites/D/Discohook/metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "description": {
+    "en": "Discohook is a free tool that allows you to personalise your server to make your server stand out from the crowd."
+  },
+  "url": "discohook.org",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/mQ5aH7h.png",
```
Comment (theusaf) [inline websites/D/Discohook/metadata.json:13]: Logos should be at least 512x512 pixels. Find/create a larger logo. You can also use an upscaling tool like waifu2x

File: websites/D/Discohook/presence.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+const presence = new Presence({
+		clientId: "1036322932932218880",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
```
Comment (theusaf) [inline websites/D/Discohook/presence.ts:7]: Imgur links are now preferred over Discord asset names

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/D/Discohook/metadata.json (hunk 1 above)
Comment (NinjaLabs-Dev) [reply websites/D/Discohook/metadata.json]: No problem, discriminator removed.

File: websites/D/Discohook/metadata.json (hunk 2 above)
Comment (NinjaLabs-Dev) [reply websites/D/Discohook/metadata.json]: Logo updated with 512x512 version

File: websites/D/Discohook/presence.ts (hunk 3 above)
Comment (NinjaLabs-Dev) [reply websites/D/Discohook/presence.ts]: Switched from asset to Imgur URL

Comment (theusaf) [reply websites/D/Discohook/presence.ts]: Make sure to mark these as resolved when you've completed them.

File: websites/D/Discohook/metadata.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+	"author": {
+		"id": "150987570715688960",
+		"name": "Ninja"
+	},
+	"service": "Discohook",
+	"description": {
+		"en": "Discohook is a free tool that allows you to personalise your server to make your server stand out from the crowd."
+	},
+	"url": "discohook.org",
+	"version": "1.1.0",
```
Comment (theusaf) [inline websites/D/Discohook/metadata.json:12]: New presences should be at `1.0.0`
```suggestion
	"version": "1.0.0",
```

File: websites/A/Apple/metadata.json [hunk 5]
```diff
@@ -90,4 +90,4 @@
 			"value": false
 		}
 	]
-}
+}
```
Comment (theusaf) [inline websites/A/Apple/metadata.json:93]: Ensure you remove the changes made to the Apple and Stake metadata files, which were likely inadvertently changed when sorting your metadata. This is causing the "version bump" problems.

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/D/Discohook/metadata.json (hunk 4 above)
Comment (NinjaLabs-Dev) [reply websites/D/Discohook/metadata.json]: This has been updated

File: websites/A/Apple/metadata.json (hunk 5 above)
Comment (NinjaLabs-Dev) [reply websites/A/Apple/metadata.json:93]: I can't actually see what has actually been changed. It was a `yarn format`, could you advice what needs changing back

Comment (theusaf) [reply websites/A/Apple/metadata.json:93]: If you take a look at the files tab of this PR, the metadata.json files for the apple and stake metadata files removed the EOF newline character.

Comment (NinjaLabs-Dev) [reply websites/A/Apple/metadata.json]: Fixed, sorry for the confusion 😅

**Comment by NinjaLabs-Dev** [comment]
Also, could I request the `hacktoberfest-accepted` label is added please 😄

**Comment by theusaf** [comment]
> Also, could I request the `hacktoberfest-accepted` label is added please 😄

This repo is a part of Hacktoberfest. If your PR gets merged, it should enter the waiting state in the Hacktoberfest dashboard automatically

**Comment by NinjaLabs-Dev** [comment]
No problem

**Review by EncryptedDev — APPROVED** [review]

**Comment by EncryptedDev** [comment]
> Also, could I request the `hacktoberfest-accepted` label is added please 😄

Yeah, as explained by theusaf above, we have added the `hacktoberfest` label to the repo to allow everyone's contributions to be valid instead of labeling presences individually (we get ~25 per week which is hard to keep up with labeling all the time)

**Comment by NinjaLabs-Dev** [comment]
Yeah no problem, wasn't 100% sure. Thanks

**Review by theusaf — APPROVED** [review]

**Comment by EncryptedDev** [comment]
You don't have to update your branch CI and files related to this PR haven't been updated

**Review by Bas950 — APPROVED** [review]

**Pull request merged**